waggle-dance-core/src/test/java/com/hotels/bdp/waggledance/yaml/AdvancedPropertyUtilsTest.java

```diff
@@ -54,48 +55,87 @@ public void setTransientProperty(char transientProperty) {
     }
   }
 
-  private final AdvancedPropertyUtils properyUtils = new AdvancedPropertyUtils();
+  private static class UnwriteableTestBean {
+    private String longPropertyName;
+
+    public String getLongPropertyName() {
+      return longPropertyName;
+    }
+  }
+
+  private static class UnreadableTestBean {
+    private String longPropertyName;
+    private char transientProperty;
+
+    @Transient
```

Comment:
Why the need for this annotation? Don't you just need a bean which has a setter with no getter? i.e. the "inverse" of the bean above.

Reply:
You're right. I don't know why I left the `@Transient` there, but I've removed it now.